internal/test/windows/windows.go

```
@@ -81,6 +87,45 @@ func (w *Windows) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+// CopyDirectory copies the files from the directory on the local host to the remote Windows VM
+// This does not copy nested directories
+func (w *Windows) CopyDirectory(localDir string, remoteDir string) error {
+	if w.SSHClient == nil {
+		return fmt.Errorf("cannot retrieve remote directory without a ssh client")
+	}
+
+	sftp, err := sftp.NewClient(w.SSHClient)
+	if err != nil {
+		return fmt.Errorf("sftp initialization failed: %v", err)
+	}
+	defer sftp.Close()
+
+	log.Printf("Copying %v directory to Windows : %v", localDir, remoteDir)
+
+	// creating a local directory to store the files and directories from remote directory.
+	err = sftp.MkdirAll(remoteDir)
+	if err != nil {
+		return fmt.Errorf("could not create %s: %v", remoteDir, err)
+	}
+
+	// Get the list of all files in the directory
+	localFiles, err := ioutil.ReadDir(localDir)
+	if err != nil {
+		return fmt.Errorf("error opening local directory %s: %v", localDir, err)
+	}
+
+	for _, localFile := range localFiles {
+		localPath := filepath.Join(localDir, localFile.Name())
+		// transfer only files and not directories
+		if !localFile.IsDir() {
+			if err = w.CopyFile(localPath, remoteDir); err != nil {
+				log.Printf("error while copying %s file to Windows : %v", localPath, err)
```

This should return an error, as that file could be expected later